HW_15/test_human.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+
+class TestHuman:
+
+
+    @pytest.mark.positive
+    def test_increase_age(self, create_human):
+        human = create_human
+        human.grow()
+        actual_age = human.age
+        expected_age = 26

```

Comment:
it is magic (I do not know why 26) number. Much better to calculate an expected result. before grow. 
expected_age = human.age + 1